packages/databricks-vscode/src/telemetry/index.ts

```diff
@@ -0,0 +1,144 @@
+import TelemetryReporter from "@vscode/extension-telemetry";
+import {DatabricksWorkspace} from "../configuration/DatabricksWorkspace";
+import {isDevExtension} from "../utils/developmentUtils";
+import {
+    DEV_APP_INSIGHTS_CONFIGURATION_STRING,
+    EventType,
+    EventTypes,
+    PROD_APP_INSIGHTS_CONFIGURATION_STRING,
+} from "./constants";
+import crypto from "crypto";
+import bcrypt from "bcrypt";
```

Comment:
Is `bcrypt` a binary dependency? If se we might have to check if our Windows builds still work.